tests/math/test_functions.py

```diff
@@ -90,6 +90,26 @@ def test_return_numpy_box(self):
         assert res.interface == "numpy"
 
 
+test_abs_data = [
+    (1, -2, 3),
+    [1, -2, 3],
+    onp.array([1, -2, 3]),
+    np.array([1, -2, 3]),
+    torch.tensor([1, -2, 3]),
+    tf.Variable([1, -2, 3]),
+    tf.constant([1, -2, 3]),
+]
+
+
+
+@pytest.mark.parametrize("t", test_abs_data)
+def test_abs(t):
+    """Test that the absolute function works for a variety
+    of input"""
+    res = fn.abs_(t)
+    assert fn.allequal(res, [1, 2, 3])

```

Comment:
I think it would be important to test complex values...

Reply:
Oh very good point!

Reply:
Done